sql/driver/conn.go

```diff
@@ -103,55 +76,13 @@ func (c *conn) exec(stmt parser.Statement, args []driver.Value) (driver.Result,
 	return driver.RowsAffected(len(rows.rows)), nil
 }
 
-func (c *conn) query(stmt parser.Statement, args []driver.Value) (*rows, error) {
+func (c *conn) query(stmt string, args []driver.Value) (*rows, error) {
 	// TODO(pmattis): Apply the args to the statement.
```

Comment:
Change this TODO to talk about sending the args in the request. Btw, after this change goes in, applying the args to the parsed statement is a good next task.